src/nimdow.nim

```diff
@@ -15,7 +15,8 @@ when isMainModule:
     if params.len == 1:
       let param = params[0].string
       if param == "-v" or param == "--version":
-        quit "Nimdow v0.5.6"
+        echo "Nimdow v0.5.6"
+        quit 0
```

Comment:
Change this to just `quit` and I'll merge

Reply:
Isn't `quit(QuitSuccess)` more appropriate? https://nim-lang.org/docs/system.html#QuitSuccess

Reply:
It would be, but `QuitSuccess` is the default value https://nim-lang.org/docs/system.html#quit%2Cint